src/aat/resources/features/F-103/F-103_Test_Data_Base.td.json

```diff
@@ -0,0 +1,16 @@
+{
+  "_guid_": "F-103_Test_Data_Base",
+  "productName": "CCD Data Store",
+  "operationName": "Retrieve audit events by case ID",
+  "method": "GET",
+  "uri": "/cases/{caseId}/events",
```

Comment:
Can you change **caseId** to **cid**?, to keep it consistent across the framework.